openfisca_us/variables/gov/states/ut/tax/income/credits/taxpayer_credit/ut_taxpayer_credit_max.py

```diff
@@ -2,6 +2,13 @@
 
 
 class ut_taxpayer_credit_max(Variable):
+    """
+    This variable computes the Initial credit before phase-out (line 16 of Utah
```

Comment:
We haven't used docstrings in variables before, but I think it's a good idea here since we're using `documentation` to show single-line descriptions in PolicyEngine.

Reply:
Is there another attribute where we could fit a long description? If not could we file an issue for it? Could be nice to surface in the api explorer.

Reply:
We can still surface docstrings in the API explorer (since it's stored as a string in `ut_taxpayer_credit_max.__doc__`), I think that's fine. But maybe something like `long_description`?